<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="GLWidget" name="glwidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolBox" name="settingBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="colorBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>200</width>
           <height>378</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Controls</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>191</width>
            <height>131</height>
           </rect>
          </property>
          <property name="title">
           <string>Rotations</string>
          </property>
          <widget class="QSlider" name="xRotSlider">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QSpinBox" name="xRotSpin">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>30</y>
             <width>47</width>
             <height>24</height>
            </rect>
           </property>
          </widget>
          <widget class="QSlider" name="yRotSlider">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QSlider" name="zRotSlider">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QSpinBox" name="yRotSpin">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>60</y>
             <width>47</width>
             <height>24</height>
            </rect>
           </property>
          </widget>
          <widget class="QSpinBox" name="zRotSpin">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>90</y>
             <width>47</width>
             <height>24</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>140</y>
            <width>191</width>
            <height>131</height>
           </rect>
          </property>
          <property name="title">
           <string>Colors</string>
          </property>
          <widget class="QSlider" name="redSlider">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QSpinBox" name="redSpin">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>30</y>
             <width>47</width>
             <height>24</height>
            </rect>
           </property>
          </widget>
          <widget class="QSlider" name="greenSlider">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QSlider" name="blueSlider">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QSpinBox" name="greenSpin">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>60</y>
             <width>47</width>
             <height>24</height>
            </rect>
           </property>
          </widget>
          <widget class="QSpinBox" name="blueSpin">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>90</y>
             <width>47</width>
             <height>24</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>glwidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
